features/admin.server-configurations.v1/forms/password-recovery-form/password-recovery-form.tsx

```diff
@@ -0,0 +1,802 @@
+/**
+ * Copyright (c) 2021-2023, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import Chip from "@oxygen-ui/react/Chip";
+import { TestableComponentInterface } from "@wso2is/core/models";
+import { CommonUtils } from "@wso2is/core/utils";
+import { Field, Form } from "@wso2is/form";
```

Comment:
Use `FinalForm` without using the `Form`

Reply:
this is also previous implementation detected as new due to file move.